core/blockchain.go

```diff
@@ -72,8 +71,8 @@ var (
 	blockValidationTimer = metrics.NewRegisteredTimer("chain/validation", nil)
 	blockExecutionTimer  = metrics.NewRegisteredTimer("chain/execution", nil)
 	blockWriteTimer      = metrics.NewRegisteredTimer("chain/write", nil)
-	blockReorgAddMeter   = metrics.NewRegisteredMeter("chain/reorg/drop", nil)
-	blockReorgDropMeter  = metrics.NewRegisteredMeter("chain/reorg/add", nil)
+	blockReorgAddMeter   = metrics.NewRegisteredMeter("chain/reorg/add", nil)
```

Comment:
nice catch!

Reply:
removing the reorg func, this is not used anymore and the lint fails, so I've removed it